influxql/task_manager.go

```diff
@@ -95,15 +95,15 @@ func (t *TaskManager) executeShowQueriesStatement(q *ShowQueriesStatement) (mode
 	for id, qi := range t.queries {
 		d := now.Sub(qi.startTime)
 
-		var ds string
-		if d == 0 {
-			ds = "0s"
-		} else if d < time.Second {
-			ds = fmt.Sprintf("%du", d)
-		} else {
-			ds = (d - (d % time.Second)).String()
+		if d >= time.Second {
```

Comment:
It would be [idiomatic](https://golang.org/doc/effective_go.html#switch) to write this as:

```go
switch {
  case d >= time.Second:
      d = d - (d % time.Second)
  case d >= time.Millisecond:
      d = d - (d % time.Millisecond)
  case d >= time.Microsecond:
      d = d - (d % time.Microsecond)
}
```

but it's not a blocker for me 👍